projects/experimental/directives/progress-segmented/progress-segmented.directive.ts

```diff
@@ -0,0 +1,27 @@
+import {Directive, HostBinding, Inject, Input} from '@angular/core';
+import {TuiDirectiveStylesService} from '@taiga-ui/cdk';
+
+import {TuiProgressSegmentedComponent} from './progress-segmented.component';
+
+@Directive({
+    selector: '[tuiProgressBar][segments]',
+    host: {
+        class: '_segmented',
+        '[attr.new]': '""', // TODO: drop in v4.0
+    },
+})
+export class TuiProgressSegmentedDirective {
+    @Input()
+    segments = 1;
+
+    @HostBinding('style.--t-segment-width')
```

Comment:
Since you already have `host` you can move it there and drop getter and decorator import.

Reply:
✅